code/modules/clothing/under/costume.dm

```diff
@@ -253,3 +253,24 @@
 	strip_delay = 50
 	sensor_mode = SENSOR_COORDS
 	random_sensor = FALSE
+
+/obj/item/clothing/under/costume/jackbros
+	name = "jack bros outfit"
+	desc = "For when it's time to hee some hos."
+	icon_state = "JackFrostUniform"
+	item_state = "JackFrostUniform"
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/yakuza
+	name = "tojo clan pants"
+	desc = "For those long nights under the traffic cone."
+	icon_state = "MajimaPants"
+	item_state = "MajimaPants"
+	can_adjust = FALSE
+
+/obj/item/clothing/under/costume/dutch
+	name = "dutch's suit"
+	desc = "You can feel a <b>god damn plan</b> coming on."
```

Comment:
should this be "god damn", "goddamn", or something else?

Reply:
Should God be capitalized?